cluster-sync/sync.sh

```diff
@@ -181,10 +208,22 @@ if [ "${CDI_SYNC}" == "test-infra" ]; then
   _kubectl apply -f "./_out/manifests/sample-populator.yaml"
   _kubectl apply -f "./_out/manifests/uploadproxy-nodeport.yaml"
 
+  # In the kind cluster, registry-populate need more capability to use buildah
```

Comment:
How come we need this twice in the sync script?

Reply:
I add the script every time, `_kubectl apply -f "./_out/manifests/registry-host.yaml", since after applied new registry-host.yaml, we need to reset the capabilities for registry-populate.
`